ui/admin-portal/src/app/components/search/define-search/define-search.component.ts

```diff
@@ -324,6 +324,11 @@ export class DefineSearchComponent implements OnInit, OnChanges, OnDestroy {
     request.sortFields = [this.sortField];
     request.sortDirection = this.sortDirection;
 
+    // Converts empty string to null for Referrer input - fixes bug whereby search included "" criterion
```

Comment:
I think that this will fix the problem - as you say - but probably not the best way to do it.

I think it would be better to fix the issue on the Spring server - to handle the special case where empty strings are passed down. In fact, we do already have some code there handling that case for certain fields - just not this one.

We can go though it together in our call tomorrow.